es7-persistence/build.gradle

```diff
@@ -0,0 +1,62 @@
+/*
+ *  Copyright 2021 Netflix, Inc.
+ *  <p>
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ *  the License. You may obtain a copy of the License at
+ *  <p>
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  <p>
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations under the License.
+ */
+
+plugins {
+    id 'com.github.johnrengelman.shadow' version '6.1.0'
+    id 'java'
+}
+
+configurations {
+    // Prevent shaded dependencies from being published, while keeping them available to tests
+    shadow.extendsFrom compileOnly
+    testRuntime.extendsFrom compileOnly
+}
+
+ext['elasticsearch.version'] = revElasticSearch7
```

Comment:
This is not necessary here as the dependency is shaded.

Reply:
please, see [previous comment](https://github.com/Netflix/conductor/pull/2324#discussion_r680390401) about a preferable way to avoid transitive dependencies conflicts.

Reply:
[Responded](https://github.com/Netflix/conductor/pull/2324#discussion_r681188522)

Reply:
I left ext['elasticsearch.version'] = revElasticSearch7 for es7-persistence module only to avoid transitive dependencies conflicts